code/controllers/subsystem/time_track.dm

```diff
@@ -35,5 +35,7 @@ SUBSYSTEM_DEF(time_track)
 	last_tick_realtime = current_realtime
 	last_tick_byond_time = current_byondtime
 	last_tick_tickcount = current_tickcount
-
-
+	SSblackbox.add_details("time_dilation_current", time_dilation_current)
+	SSblackbox.add_details("time_dilation_avg_fast", time_dilation_avg_fast)
+	SSblackbox.add_details("time_dilation_avg", time_dilation_avg)
+	SSblackbox.add_details("time_dilation_avg_slow", time_dilation_avg_slow)
```

Comment:
This way too many stats. Change it to just this

```dm
SSblackbox.set_details("time_dilation", time_dilation_avg_slow)
```

Reply:
I don't agree, there's no such thing as too many stats.

Reply:
well if we are going to use details and not value, we could merge it up into one.

Also jordie was looking at making this support json and versioning, so we could wait for that and then that would simplify it

Reply:
`set_details` would overwrite the value each call.
I think a running track of data like this would be better suited to its own table rather than concatenated together but otherwise it'd defs be better as json.